apps/frontend/src/App.tsx

```diff
@@ -78,61 +79,63 @@ const App: FC<Record<string, never>> = () => {
   }, [setCurrentOverrides]);
 
   return (
-    <Frame title="ASAP Hub">
-      <GoogleTagManager containerId={GTM_CONTAINER_ID} />
-      <AuthProvider>
-        <SentryAuth0 />
-        <Router history={history}>
-          <LastLocationProvider>
-            <Frame title={null}>
-              <Switch>
-                <Route path={welcome.template}>
-                  <UtilityBar>
-                    <ToastStack>
-                      <Welcome />
-                    </ToastStack>
-                  </UtilityBar>
-                </Route>
-                <Route path={logout.template}>
-                  <Frame title="Logout">
-                    <Logout />
-                  </Frame>
-                </Route>
-                <Route exact path={staticPages({}).terms.template}>
-                  <BasicLayout>
-                    <Frame title={null}>
-                      <Content pageId="terms-and-conditions" />
+    <RecoilRoot>
```

Comment:
Have we removed RecoilRoot from the rest of the application (excluding tests) as it will now be available everywhere?

Reply:
I think so @kmaid. I removed it from `AuthenticatedApp` component
See this commit https://github.com/yldio/asap-hub/pull/1407/commits/443b1d184098d02ab1d4beb2853e02f96879a1e9